<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Startup</class>
 <widget class="QWidget" name="Startup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CleverPocket</string>
  </property>
  <widget class="QWidget" name="Startup" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1360</width>
     <height>768</height>
    </rect>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1360</width>
      <height>768</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="loginPage">
     <widget class="QGroupBox" name="loginGroupBox">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>240</y>
        <width>300</width>
        <height>210</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLineEdit" name="usernameLineEdit">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>200</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="placeholderText">
        <string>Username</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="passwordLineEdit">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>90</y>
         <width>200</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QPushButton" name="loginPushButton">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>150</y>
         <width>200</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </widget>
     <widget class="QCheckBox" name="keeploggedCheckBox">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>460</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
       </font>
      </property>
      <property name="text">
       <string>Keep me logged in</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="forgotPasswordLinkButton">
      <property name="geometry">
       <rect>
        <x>475</x>
        <y>460</y>
        <width>150</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Forgot your password?</string>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="registerLinkButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>540</y>
        <width>160</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Not yet registered?</string>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="labelLogoImage">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>30</y>
        <width>301</width>
        <height>201</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>exemplulogo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="registerPage">
     <widget class="QGroupBox" name="registerGroupBox">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>84</y>
        <width>500</width>
        <height>600</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLineEdit" name="firstNameLineEdit">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>190</width>
         <height>30</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>First Name</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="usernameRLineEdit">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>100</y>
         <width>400</width>
         <height>30</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Username</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lastNameLineEdit">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>40</y>
         <width>190</width>
         <height>30</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Last Name</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="passwordRLineEdit">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>220</y>
         <width>400</width>
         <height>30</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="confirmPasswordRLineEdit">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>280</y>
         <width>400</width>
         <height>30</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Confirm Password</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="emailLineEdit">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>160</y>
         <width>400</width>
         <height>30</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Email</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="phoneLineEdit">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>400</y>
         <width>400</width>
         <height>30</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Phone Number</string>
       </property>
      </widget>
      <widget class="QComboBox" name="countryComboBox">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>340</y>
         <width>400</width>
         <height>30</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Country</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tara 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tara 2</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="registerLinkButton_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>490</y>
         <width>400</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>Register</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="termsAndConditonsCheckBox">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>440</y>
         <width>121</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Agree to the</string>
       </property>
      </widget>
      <widget class="QCommandLinkButton" name="termsAndConditionsLinkButton">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>432</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Terms and Conditions</string>
       </property>
       <property name="iconSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCommandLinkButton" name="alreadyRegisteredLinkButton">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>550</y>
         <width>160</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Already have an account?</string>
       </property>
       <property name="iconSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="termsAndConditionPage"/>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="startup.qrc"/>
 </resources>
 <connections/>
</ui>
